pkg/rpcserver/server_test.go

```diff
@@ -39,7 +40,9 @@ func TestServer(t *testing.T) {
 	}
 	h := new(hello)
 	h.cmd = srv.Register(h.process)
-	srv.Run()
+	if err := srv.Run(); err != nil {
+		panic(err)
```

Comment:
Replace `panic` with `require.NoError`

Reply:
This is the format of goimports, which I think should be used consistently